src/Native/Unix/CMakeLists.txt

```diff
@@ -231,3 +231,7 @@ add_subdirectory(System.Native)
 add_subdirectory(System.Net.Http.Native)
 add_subdirectory(System.Net.Security.Native)
 add_subdirectory(System.Security.Cryptography.Native)
+
+if(OSX)
+add_subdirectory(System.Security.Cryptography.Native.Apple)
```

Comment:
Sure.  I feel like I ran across a left-aligned conditional subdirectory in a different file; because in my head I had the "you're copying style, ignore the fact that you don't like it" bit set.  But since I can't find it now, I'm happy to change.